controllers/bpfman-operator/configmap.go

```diff
@@ -43,12 +44,15 @@ import (
 // +kubebuilder:rbac:groups=apps,resources=daemonsets,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;list;watch;create
 // +kubebuilder:rbac:groups=storage.k8s.io,resources=csidrivers,verbs=get;list;watch;create;delete
+// +kubebuilder:rbac:groups=security.openshift.io,resources=securitycontextconstraints,verbs=get;list;watch;create;delete
 // +kubebuilder:rbac:groups=bpfman.io,resources=configmaps/finalizers,verbs=update
 
 type BpfmanConfigReconciler struct {
 	ReconcilerCommon
 	BpfmanStandardDeployment string
 	CsiDriverDeployment      string
+	RestrictedSCC            string
+	IsOpenshift              bool
```

Comment:
This struct is created in [configmap_test.go](https://github.com/bpfman/bpfman-operator/blob/main/controllers/bpfman-operator/configmap_test.go#L87), these new variables should be initialized there?

Reply:
we _should_ have a unit test there for this functionality :/ I'm going to make a follow up issue for this

Reply:
https://github.com/bpfman/bpfman-operator/issues/27